app/views/users/_ux.html.erb

```diff
@@ -1,6 +1,6 @@
 <%= javascript_packs_with_chunks_tag "stickySaveFooter", defer: true %>
 
-<%= form_for @user, html: { id: "ux-customization-form", class: "sticky-footer-form grid gap-6" } do |f| %>
+<%= form_for @user, remote: true, authenticity_token: true, html: { id: "ux-customization-form", class: "sticky-footer-form grid gap-6" } do |f| %>
```

Comment:
I think you can replace by manually using `sendFetch` on save https://github.com/forem/forem/blob/master/app/assets/javascripts/utilities/sendFetch.js